.github/workflows/on_pr_push_master.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Super-Linter
-        uses: github/super-linter@latest
+        uses: github/super-linter@v3.13.1
```

Comment:
Not sure that it will help due to this issue https://github.com/github/super-linter/issues/708

Reply:
Tried to fix accordingly to a comment about v3.13.1: https://github.com/github/super-linter/issues/938

Reply:
Here's the comment btw: https://github.com/github/super-linter/issues/938#issuecomment-720769293

Reply:
For now, you can use @Neha-Sinha2305 fork https://github.com/Neha-Sinha2305/super-linter